.github/workflows/htmltest.yml

```diff
@@ -0,0 +1,59 @@
+# .github/workflows/htmltest.yml
+# Run htmltest link checker on generated HTML output
+name: Link check with htmltest
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - trunk
+  pull_request:
+
+jobs:
+  htmltest:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all history for .GitInfo and .Lastmod
```

Comment:
I don't think we need to check the whole history to check links.

Reply:
> I don't think we need to check the whole history to check links.

That's very true. I removed the with clause, this certainly will make checkout faster.
You may have to adapt the link checker's configuration file according to your needs, too.